mautic/mautic
# fixing empty tag segmentation

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✔️ 
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | No
| Related user documentation PR URL      | Not relevant
| Related developer documentation PR URL | Not relevant
| Issue(s) addressed                     | Fixes #11178



#### Description:


Create a segment with empty/not empty tag, see no contact matches criteria

#### Steps to test this PR:


1. Create segment with empty/not empty tag,
2. See it works ✔️ 




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11179"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
